### PR TITLE
Update docu after migration to go-gatekeeper #5

### PR DESCRIPTION
**Pull request by p53**
# Title

## Type

[] Bug fix
[] Feature request
[] Enhancement
[] Docs

## Checklist:

- [ ] My change requires a change to the documentation or CHANGELOG.
- [ ] I have updated the documentation/CHANGELOG accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,27 @@
-# Louketo Proxy
+# Gatekeeper Proxy
```
Comment (ackerleytng) [inline README.md:1]: Shall we stick with "Gatekeeper"? So perhaps change all references of "Gatekeeper Proxy" to "Gatekeeper"

File: docs/building.md [hunk 2]
```diff
@@ -4,29 +4,29 @@ Ensure you have Golang 1.11 (or newer) and Git installed
...
 First clone the Louketo Proxy repository:
-    
-    git clone https://github.com/louketo/louketo-proxy.git
-    cd louketo-proxy
-    
+
+    git clone https://github.com/go-gatekeeper/gatekeeper.git
+    cd gatekeeper
+
 To build Louketo Proxy run:
```
Comment (ackerleytng) [inline docs/building.md:13]: Missed reference to Louketo!

File: README.md (hunk 1 above)
Comment (p53) [reply README.md:1]: it is still proxy...

File: docs/building.md (hunk 2 above)
Comment (p53) [reply docs/building.md]: fixed

**Comment by p53** [comment]
related to #5

**Pull request merged**